bin/cylc-depend

```diff
@@ -61,18 +61,10 @@ else:
 if not TaskID.is_valid_id(task_id):
     sys.exit("Invalid task ID: " + task_id)
 
-try:
-    proxy = cylc_pyro_client.client( suite, pphrase, options.owner,
-            options.host, options.pyro_timeout,
-            options.port ).get_proxy( 'command-interface' )
-    prompt( 'Add new prerequisite to ' + task_id + ' in ' + suite, options.force )
-    result = proxy.put( 'add prerequisite', task_id, msg )
-except Exception, x:
-    if cylc.flags.debug:
-        raise
-    sys.exit(x)
-
-if result[0]:
-    print result[1]
-else:
-    sys.exit( result[1] )
+prompt('Add new prerequisite to %s in %s' % (task_id, suite), options.force)
+pclient = SuiteCommandClient(
+    suite, pphrase, options.owner, options.host, options.pyro_timeout,
+    options.port)
+pclient.put_command('add_prerequisite', task_id, msg)
```

Comment:
This change and the others are nice. Hopefully this is close to the kind of interface we'll need for the generic communications refactor.